kedify-agent/values.yaml

```diff
@@ -17,8 +17,8 @@ agent:
   createKedifyConfiguration: true
 
   image:
-    tag: "v0.0.53"
-    repository: ghcr.io/kedify/agent
+    tag: "1729243540"
+    repository: wozniakjan/agent
```

Comment:
should this go in?

Reply:
ah, right, shouldn't go in, just residue from testing https://github.com/josefkarasek/agent/pull/3